.env.example

```
@@ -9,20 +9,30 @@ OPENAI_API_KEY=
+# LLM for evaluation (optional - leave blank to use defaults)
+EVAL_LLM_MODEL=
+EVAL_LLM_PROVIDER=
+EVAL_LLM_PROVIDER_BASE_URL=
+
 # Langsmith Configuration for tracing (optional)
 LANGSMITH_TRACING=
 LANGSMITH_ENDPOINT=
 LANGSMITH_API_KEY=
 LANGSMITH_PROJECT=
+
+EVAL_JURY_JUDGES=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)